CMake/external_pybind11.cmake

```diff
@@ -84,3 +81,19 @@ endfunction()
 
 # Trigger the build
 get_pybind11()
+
+# This function override "pybind11_add_module" function,  arguments is same as "pybind11_add_module" arguments
+# pybind11_add_module(<name> SHARED [file, file2, ...] )
+# Must be declared after pybind11 configuration above
+# "_pybind11_add_module" is calling the origin pybind11 function
+function( pybind11_add_module _arg_project_name _arg_library_type ...)
+
+    message( STATUS "adding python module --> ${_arg_project_name}" )
+    _pybind11_add_module( ${ARGV})
+
+    # Force Pybind11 not to share pyrealsense2 resources with other pybind modules.
+    # With this definition we force the ABI version to be unique and not risk crashes on different modules.
+    # (workaround for RS5-10582; see also https://github.com/pybind/pybind11/issues/2898)
```

Comment:
Can we add a note?
`// NOTE this workaround seems to be needed for debug compilations only`

By the way:
`target_compile_definitions(${project_name} PRIVATE "$<$<CONFIG:DEBUG>:PYBIND11_COMPILER_TYPE=_${project_name}_abi\">")`
Will work on in debug?

Reply:
Note will be added.
About the debug, I don't know.. It works on my small example, I don't know if it will work on a more complex usage so I prefer not to share the resources between modules, each one is self contained.